src/App.tsx

```diff
@@ -1,24 +1,176 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import * as todoService from './services/todos';
+import { Todo } from './types/Todo';
+import { TodoError } from './components/TodoError/TodoError';
+import { Error } from './types/Error';
+import { Filter } from './types/Filter';
 
-const USER_ID = 0;
+const USER_ID = 12007;
+
+const preparedTodos = (todosList: Todo[], selectedFilter: Filter): Todo[] => {
```

Comment:
The function name must begin with a verb
```suggestion
const prepareTodos = (todosList: Todo[], selectedFilter: Filter): Todo[] => {
```